style.css

```diff
@@ -1,36 +1,860 @@
+  }
+
+  .section {
+    background: url(Img/mainbg.webp);
+    background-repeat: no-repeat;
+    background-size: cover;
+    height: max-content;
+    display: flex;
+    flex-direction: column;
+    overflow-y: hidden;
+    overflow-x: hidden;
+  }
+
+  .section-container {
+    display: flex;
+    width: 80%;
+    padding-top: 4%;
+    margin-left: 10%;
+    margin-right: 25%;
+    margin-top: 5%;
+  }
+
+  .program-box {
+    display: flex;
+    flex-direction: column;
+    margin-left: 10px;
+    background: rgba(121, 121, 121, 0.3);
+    width: 100%;
+    font-family: "Lato", sans-serif;
+  }
+
+  .program-icon {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+  }
+
+  .program-icon h3 {
+    font-size: 20px;
+    color: orangered;
+  }
+
+  .program-icon img {
+    width: 35%;
+  }
+
+  .program-box p {
+    color: white;
+    padding: 3%;
+    font-size: 15px;
+  }
+
+  .title-box {
+    font-family: "Lato", sans-serif;
+    margin-left: 35%;
+    padding-top: 5%;
+    margin-bottom: -55px;
+    color: white;
+  }
+
+  .join {
+    display: none;
+  }
+
+  #Program a {
+    color: white;
+    margin-left: 30%;
+    font-size: 20px;
+    margin-top: 5%;
+    text-decoration: underline;
+  }
+
+  .speakers {
+    background-color: #fff;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    text-align: center;
+    height: 100vh;
+  }
+
+  .speakers h2 {
+    font-size: 30px;
+    margin-top: 30px;
+    font-weight: 400;
+    color: #272a31;
+    text-align: center;
+  }
+
+  .featured-speak {
+    display: grid;
+    grid-template-columns: 1fr;
+    padding: 30px 0;
+    gap: 10px;
+    text-align: left;
+  }
+
+  .featured-speakers {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    gap: 15px;
+  }
+
+  .featured-speakers .grid {
+    display: grid;
+    grid-template-columns: 2fr 4fr;
+    gap: 5px;
+  }
+
+  .grid h2 {
+    font-size: 10px;
+    font-weight: 700;
+    color: red;
+  }
+
+  .featured-speakers ul {
+    text-decoration: none;
+    list-style: none;
+  }
+
+  .featured-speakers2 {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    gap: 15px;
+  }
+
+  .featured-speakers2 .grid {
+    display: grid;
+    grid-template-columns: 2fr 4fr;
+    gap: 5px;
+  }
+
+  .container3 {
+    max-width: 1100px;
+    margin: 0 auto;
+    padding: 0 20px;
+  }
+
+  hr.line2 {
+    border: 1px solid orangered;
+    width: 5%;
+    margin: auto;
+    margin-top: 20px;
+  }
+
+  .featured-speakers2 ul {
+    text-decoration: none;
+    list-style: none;
+  }
+
+  .featured-speakers2 ul h2 {
+    font-size: 24px;
+    font-weight: 700;
+    color: #272a31;
+  }
+
+  .featured-speakers2 .p1 {
+    font-size: 10px;
+    font-weight: 700;
+    color: orangered;
+    text-align: left;
+    padding-top: 0;
+  }
+
+  .featured-speakers2 .p2 {
+    font-size: 10px;
+    font-weight: 700;
+    color: #272a31;
+    text-align: left;
+    padding-top: 10px;
```

Comment:
- [ ] If you notice carefully, the speaker information are italic. Kindly apply italic property on them to match the original design ☺️.
- [ ] Again, please ensure that the colors match the ones of the original design. The color used for this case is `#d3d3d3` and `#ec5242`
- [ ] Please also ensure to provide the title of the missing speaker.
- [ ] In addition, kindly ensure to stick to the font families that have been provided in the original design. We have been given `cocogoose` and `lato` and therefore we should stick to those throughout our application design ☺️ 

Your design 😎 
![image](https://github.com/Zel-hub7/Capstone-project1/assets/99394374/eef308ce-bf2d-48dd-bc91-9ec021d00e63)

Original design 😎 
![image](https://github.com/Zel-hub7/Capstone-project1/assets/99394374/fc22a293-37b5-462a-8e2e-d43fef85606a)